Under construction

2bleO opened the pull request:
In the Pull Request we:

Createa and style Home Page
Create and style about page
Pilots section created dynamically

2bleO committed:
Repo Set Up

Added Linters and setup files

2bleO committed:
Create index.html

2bleO committed:
Update package-lock.json

2bleO committed:
Modify index.html

Mobile Home page HTML is ready

2bleO committed:
Style mobile home page

2bleO committed:
Create Pilots section

Pilot section created dynamically in main.js.

Necessary images for home page uploaded

2bleO committed:
Update index.html

fix style link

2bleO committed:
Update index.html

Fix Partner Logos route

2bleO committed:
Update README

2bleO committed:
Fix initial zoom Issue

set body width to 100%

2bleO committed:
Revert "Fix initial zoom Issue"

This reverts commit 10aa7913b4e32a6dfc25660ff24783505548b351.

2bleO committed:
Revert "Update README"

This reverts commit a263a7262c85e3f2ab0fa27b4868e48ed5bc6343.

2bleO committed:
Update body width

2bleO committed:
Update README.md

2bleO committed:
Update README.md

2bleO committed:
Create screenshot.png

2bleO committed:
modify Body min width

2bleO committed:
Home page Fully Responsive

Add media queries and adapt bootstrap classes for a fully responsive home page

2bleO committed:
Update linters.yml

2bleO committed:
Create and style About page

2bleO committed:
Modify Files Routes

2bleO committed:
Update File Path

2bleO committed:
Modify File Path

2bleO committed:
Fix Desktop version footer padding

2bleO committed:
Update about.html

Fix link to home page

2bleO committed:
Update about.html

Fix Partners Logos File Path

2bleO committed:
Update README.md

Add Explanatory Video

2bleO committed:
Update README.md

vichuge commented on README.md:
```diff
@@ -1 +1,54 @@
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+* [GitHub](https://www.github.com)
+* [Microverse](https://microverse.org)
```
- [x] Please, the Acknowledgments section should contain appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the author of the original design, as required by the [Creative Commons License](https://creativecommons.org/licenses/). 🐶 
- I could see you add it on License, and that's pretty good, I invite you to add author credits on acknowledgments and the License on License section.

vichuge commented on index.html:
```diff
@@ -0,0 +1,145 @@
+    </div>
+    <header class="row justify-content-center">
+      <nav class="navbar flex-md-row navbar-expand-md navbar-light">
+        <a class="navbar-brand d-md-flex flex-md-row-reverse" href="index.html"> SSCGR<br>2021 <i class="fas fa-car"></i></a>
```
- [x] When I click the logo on the `about` page, I got a 404 error. Please, fix it. 🚗
![image](https://user-images.githubusercontent.com/20563252/124698203-f2d9ca80-dead-11eb-8742-3c09ff6481a2.png)

vichuge commented on index.html:
```diff
@@ -0,0 +1,145 @@
+          <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
+            <ul class="navbar-nav">
+              <li class="nav-item">
+                <a class="nav-link active" aria-current="page" href="/index.html">Home</a>
```
- [x] When I click `Home` on the same home page, I got a 404 error. Please fix it 🐱 
![image](https://user-images.githubusercontent.com/20563252/124698313-38969300-deae-11eb-8df1-023cdc32ea21.png)

vichuge commented on pages/about.html:
```diff
@@ -0,0 +1,113 @@
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"/>
+</head>
+<body class="bg-light" id="body">
+  <div class="container">
```
- [x] On the `about` page, you're using margin on sides but no on the `Home` page, I invite you to remove the margin on about. 🌵 
- [x] I think for the same reason, the burger icon appears bigger on the `about` page, and the size should be the same for both pages, please fix it. 🦊 
![image](https://user-images.githubusercontent.com/20563252/124698598-cf634f80-deae-11eb-8603-5a70121e6202.png)

vichuge commented on README.md:
```diff
@@ -1 +1,54 @@
+
+> Web page to Showcase a Fictional Super SportCar Race as Microverse Capstone Project.
+
+![screenshot](/Assets/img/screenshot.png)
```
- [x] The image you choose on the readme file is bigger than required, I kindly invite you to check that, to have an image of this size could be annoying to the person who checked your project. 🐸
- I add a recommendation (ok well, two recommendations).
![image](https://user-images.githubusercontent.com/20563252/124698902-6f20dd80-deaf-11eb-830e-06c639f0be50.png)
![image](https://user-images.githubusercontent.com/20563252/124698928-7811af00-deaf-11eb-8db7-2b1763560011.png)

vichuge commented on Assets/sass/main.scss:
```diff
@@ -0,0 +1,398 @@
+}
+
+/* Media Query */
+@media (min-width: 1px) {
```
- [x] (Ok, this is a hard one) Please, the CSS styling is built as **mobile-first**, media queries are used to change the design from mobile to desktop, not the other way around. In this case, you're using the `@media` for mobile, the default style should be for mobile and the `@media` should be used for desktop, I kindly invite you to fix this. I'm sure you'll find a way to solve it as well as possible. 🦁

vichuge reviewed (changes requested):
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![mario_ladders](https://user-images.githubusercontent.com/20563252/121369265-4e418880-c901-11eb-848c-1eff06d5a4cf.gif)

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

2bleO committed:
Update index.html

2bleO committed:
Fix Nav-toggler size

2bleO committed:
Update README.md

-Fix  Acknowledgments

2bleO committed:
Update screenshot.png

2bleO committed:
Update Thumbs.db

2bleO committed:
Update about.html

-Fix About page Home page  logo link 404
-Fix About page container to container fluid

2bleO committed:
Update about.css

Fix Media Queries

2bleO committed:
Update about.scss

Fix Media Queries

2bleO committed:
Update main.css

Fix Media Queries

2bleO committed:
Update main.scss

Fix Media Queries

2bleO committed:
Update about.html

Fix Media Queries

de-scholar reviewed (approved):
Hi @2bleO , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
